### PR TITLE
ask-cli: fix subcommands

### PR DESCRIPTION
**Pull request by SMillerDev**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----
Fixes https://github.com/Homebrew/homebrew-core/issues/36444

The wrapper is needed due to weird node relative command invocation. A test was added to prevent it from breaking again.

### CODE REVIEW

File: Formula/ask-cli.rb [hunk 1]
```diff
@@ -16,11 +16,16 @@ class AskCli < Formula
 
   def install
     system "npm", "install", *Language::Node.std_npm_install_args(libexec)
-    bin.install_symlink Dir["#{libexec}/bin/*"]
+    (bin/"ask").write <<~EOS
+      \#!/bin/bash
+      #{prefix}/libexec/bin/ask $@
+    EOS
+    bin.install_symlink
```
Comment (chrmoritz) [inline Formula/ask-cli.rb]: Why didn't you use `write_exec_script` here? Also the parameterless `bin.install_symlink` looks superfluous to me.

```suggestion
    bin.write_exec_script libexec/"bin/ask"
```

Comment (SMillerDev) [reply Formula/ask-cli.rb]: Honestly, I didn't know it existed. Thanks @chrmoritz !

**Review by fxcoudert — APPROVED** [review]

**Comment by fxcoudert** [comment]
@BrewTestBot test this please

**Comment by SMillerDev** [comment]
thanks @fxcoudert !

**Pull request closed without merging**